src/main/webapp/script.js

```diff
@@ -56,8 +57,17 @@ function initMap() {
 
         setEndpoint(place);
     })
-    // Initialise visualisation
-    populateAQVisualisationData();
+    // Initialise visualisation when the bounds of the map changed.
+    // map.getBounds() is undefined until the map tiles have finished loading,
+    // at which point the bounds change
+    google.maps.event.addListener(map, 'bounds_changed', function() {
+        if (refreshAQLayerTimeout) {
```

Comment:
I realised the block indents for the javascript is in indents of 4, but the JS Google Guide says block indentation is +2 spaces? @purple-waves

Reply:
actually, am talking to Sai rn, and its all good, consistency trumps style guide and also 4 vs 2 isn't  a huge deal

Reply:
Yeah I did see that when I was having a look at some stuff earlier, iirc I had a conversation with Sai earlier about spacing, and the outcome was that it didn't matter as long as it was consistent. The 4 spacing is the result of my constant battles with the editor because it keeps changing the default indentation to 4 spaces and I couldn't get it to stop